app/jobs/replication/gcp_delivery_job.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Replication
+  # @see Replication::IbmProvider for how S3 credentials and bucket are configured
```

Comment:
What are your thoughts on whether the `IbmProvider` service is long for this world?

Reply:
tis not long for this world at all, in my estimation 🙂 

it's not yet ticketed and on the board, but removal is captured in the currently last bullet point of the checklist for swapping IBM for GCP: https://github.com/sul-dlss/preservation_catalog/issues/2237#issuecomment-2768165327

oh!  but i see, stray copypasta comment referencing it like it'll be around for a while.  good catch, will fix...